events-api/src/modules/ticket-payment/ticket-payment.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { TicketPaymentService } from './ticket-payment.service';
+import { HttpModule } from '@nestjs/axios';
+
+@Module({
+  providers: [TicketPaymentService],
+  exports: [TicketPaymentService],
+  imports: [HttpModule],
```

Comment:
You can configure http module here using register async static method.

Reply:
fixed